duckduckgo.safariextension/js/oninstall.js

```diff
@@ -27,4 +28,16 @@ safari.self.addEventListener("message", function(event) {
       }
     }, 500);
   }
+
+  if (event.name === "atb_param") {
+    var atbParam = event.message;
+    var input = document.createElement('input');
+    input.type = 'hidden';
+    input.name = 'atb';
+    input.value = atbParam;
+    var form = document.querySelector('#search_form_homepage');
```

Comment:
@bsstoner This will only add the ATB param to the homepage. In other cases it will fail. I am not sure if it makes sense to ensure that `document.querySelector` is not actually `null` as this will be executed in a sandbox, but I still did that.